README.md

```diff
@@ -47,6 +56,14 @@ You can find the core Applifting documents which we abide by below:
 4. 📜🤝📜 [**Conflict solution guide**](./en/managing-conflicts.md) - This document describes the process that Applifting uses for solving conflicts of both work and personal character
 5. 📜🏛📜 [**How Council of Elders works**](https://www.notion.so/applifting/How-Council-of-Elders-Works-8dff507d7f9247fdb68c0d4029331628) - Defines powers and responsibilities of our board. Describes the process, how council members are elected by Applifters. 
 
+## Outreach
+
+We do not stop at the borders of our company. We are committed to having a positive impact on the communities around us.
+
+* 🤗 [**Spirit of Applifting**](https://www.notion.so/applifting/Spirit-of-Applifting-1cbf1c9247cc442da73f768ffcedadc4) - Applifters can do whatever they think is helpful to others and Applifting will pay for their time.
+* 💰 **Small portion of profit to charities** - Every year, we give out 0,5% of profit to charities. Charities are selected by Applifters.
```

Comment:
"...during Christmas rewards. Every Applifter can donate to cause that's meaningful to him."